packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -33,6 +34,11 @@ export default class ComposedModal extends Component {
   endSentinel = React.createRef();
 
   static propTypes = {
+    /**
+     * Specify the aria-label for bx--modal-container
+     */
+    ariaLabel: PropTypes.string,
```

Comment:
This is definitely a point where the codebase is split, but would you want to do `aria-label` if doing the DOM attribute and `label` if we're setting it automatically for folks?

In other words, if someone is explicitly setting an ARIA label then they use `aria-label` as the prop.